be/src/storage/rowset/block_split_bloom_filter.h

```diff
@@ -61,6 +73,15 @@ class BlockSplitBloomFilter : public BloomFilter {
         }
     }
 
+    static uint32_t _get_block_index_default(uint64_t hash, uint32_t block_size) {
+        return (uint32_t)(hash >> 32) & (block_size - 1);
+    }
+    static uint32_t _get_block_index_parquet(uint64_t hash, uint32_t block_size) {
```

Comment:
If block indexer is the only difference, it seems pass the `get_block_index` as a argument is better?